### PR TITLE
Prepare PerpsV2 test deployment

### PR DESCRIPTION
**Pull request by leomassazza**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -355,6 +356,29 @@ const getFeeds = ({ network, path, fs, deploymentPath, useOvm = false } = {}) =>
 	}, {});
 };
 
+const getOffchainFeeds = ({ network, path, fs, deploymentPath, useOvm = false } = {}) => {
+	let offchainFeeds;
```
Comment (davidvuong) [inline index.js:360]: Nit: Minor but you can probably avoid this var entirely and just return in the if/else.

File: publish/deployed/local-ovm/perpsv2-markets.json [hunk 2]
```diff
@@ -17,7 +17,9 @@
 		"maxLeverage": "10",
 		"maxMarketValue": "20000000",
 		"maxFundingVelocity": "0.1",
-		"skewScale": "300000000"
+		"skewScale": "300000000",
```
Comment (davidvuong) [inline publish/deployed/local-ovm/perpsv2-markets.json:20]: I probably forgot to change these and they're just dummy values but do you mind lowering the `skewScale` and `maxMarketValue` something a bit smaller like `15000` and `1000` respectively? The values were previously in USD but now in native units. Same with pETH.

File: publish/releases.json [hunk 3]
```diff
@@ -1420,6 +1435,25 @@
...
 			"sips": [267],
 			"released": true
+		},
+		{
+			"name": "Xxxxx",
+			"version": {
+				"major": 2,
+				"minor": 79
+			},
+			"sips": [279, 280, 281, 285],
```
Comment (davidvuong) [inline publish/releases.json:1445]: 🚀

File: publish/deployed/local-ovm/perpsv2-markets.json (hunk 2 above)
Comment (leomassazza) [reply publish/deployed/local-ovm/perpsv2-markets.json:20]: Done (also removed data in sample mainnet-ovm/perpsv2-market.json, that should be empty and add markets as needed. Maintaining the file un-commited for testing release)

**Comment by davidvuong** [comment]
Oh I forgot to mention before, I spoke with Afif and we have some realistic values BTC/ETH wrt `maxMarketValue` and `skewScale`:

```
ETH
skewScale - 1M ETH
maxMarketValue - 20k ETH

BTC
skewScale - 50k BTC
maxMarketValue - 1500 BTC
```

File: test/integration/dual/staking.integration.js [hunk 4]
```diff
@@ -9,34 +9,34 @@ const { skipFeePeriod, skipMinimumStakeTime } = require('../utils/skip');
 
 const ethers = require('ethers');
 
-describe('staking & claiming integration tests (L1, L2)', () => {
+describe('staking & claiming integration tests (L1, L2)', async () => {
```
Comment (davidvuong) [inline test/integration/dual/staking.integration.js:12]: Oh how interesting. Is this why integration tests intermittently fail?

**Review by davidvuong — APPROVED** [review]
Couple of comments but LGTM

File: test/integration/dual/staking.integration.js (hunk 4 above)
Comment (leomassazza) [reply test/integration/dual/staking.integration.js:12]: that was an attempt to fix it, but still not working and in fact IIRC async keyword doesn't impact describes so was a guesswork

**Pull request merged**